packages/backend-auth/API.md

```diff
@@ -84,6 +90,11 @@ export type OidcProviderFactoryProps = Omit<OidcProviderProps, 'clientId' | 'cli
     clientSecret: BackendSecret;
 };
 
+// @public (undocumented)
+export type TriggerConfig = {
+    triggers?: Partial<Record<TriggerEvent, ConstructFactory<ResourceProvider<FunctionResources>>>>;
+};

```

Comment:
This already exists on main here https://github.com/aws-amplify/amplify-backend/blob/7398baa80461dee7d3abd47b51926c640fb6b6e8/packages/backend-auth/src/factory.ts#L26-L36 .